ceph-salt-formula/salt/ceph-salt/apply/tuned-latency.sls

```diff
@@ -0,0 +1,29 @@
+
+{% if grains['id'] in pillar.get('ceph-salt:minions:latency', []) %}
+
+install tuned:
```

Comment:
Note that in `ceph-salt` we always "fail hard" to stop execution at first error, so you have to add `- failhard: True` to all stages, see some examples:
- https://github.com/ceph/ceph-salt/blob/master/ceph-salt-formula/salt/ceph-salt/apply/software.sls#L13
- https://github.com/ceph/ceph-salt/blob/master/ceph-salt-formula/salt/ceph-salt/apply/cleanup.sls#L21